### PR TITLE
feat: added support for live preview ssr 2.0

### PR DESCRIPTION
**Pull request by contentstackMridul**
1. Add live preview hash, content type uid and entry uid query params in all the “a” tag available on user’s preview page
2. Updated readme file for onLiveEdit function

### CODE REVIEW

File: docs/live-preview-configs.md [hunk 1]
```diff
@@ -132,13 +132,11 @@ The `stackSdk` object represents the `Stack` class that we get by executing the
 
 ## `onLiveEdit(callback: () => void)`
 
-The `onLiveEdit` method modifies or alters the content inside the Live Preview panel as soon as a change is made in the entry.
+The onLiveEdit method modifies or alters the content inside the Live Preview panel as soon as a change is made in the entry. This method runs a single API request to retrieve draft content from the entry and display the changes in the Live Preview panel.
 
-> **Note:** This function only works when [`ssr`](#ssr) is set to `false`, indicating that the application is of type [CSR](https://www.contentstack.com/docs/developers/set-up-live-preview/set-up-live-preview-for-your-website/#client-side-rendering-csr-).
+> **Note:** The onLiveEdit method will not fetch the published content of the entry and is only applicable in the Client-Side Rendering ([CSR](https://www.contentstack.com/docs/developers/set-up-live-preview/set-up-live-preview-for-your-website/#client-side-rendering-csr-)) mode.
```
Comment (Deepak-Kharah) [inline docs/live-preview-configs.md:137]: Did we get this from the docs team?

File: src/live-preview.ts [hunk 2]
```diff
@@ -95,11 +95,57 @@ export default class LivePreview {
...
+        const ctUid = docUrl.searchParams.get("content_type_uid");
+        const entryUid = docUrl.searchParams.get("entry_uid");
+        if (livePreviewHash && ctUid && entryUid) {
+            newUrl.searchParams.set("live_preview", livePreviewHash);
+            newUrl.searchParams.set("content_type_uid", ctUid);
+            newUrl.searchParams.set("entry_uid", entryUid);
+        }
+        return newUrl.href;
+    }
+
```
Comment (Deepak-Kharah) [inline src/live-preview.ts:148]: Could we move it to a separate file?

**Review by Deepak-Kharah — COMMENTED** [review]
Let's see if we could incorporate these changes

File: docs/live-preview-configs.md (hunk 1 above)
Comment (contentstackMridul) [reply docs/live-preview-configs.md:137]: Yes
https://docs.google.com/document/d/15vqTPuNTYnlETPkaLsyHNuiYBWzh3fu-quH351gzwi4/edit#

**Review by Deepak-Kharah — APPROVED** [review]
Changes looks good

**Pull request merged**